client/me/account/hosting-dashboard-opt-in-form.tsx

```diff
@@ -93,6 +105,34 @@ export default function HostingDashboardOptInForm() {
 							<span>{ translate( 'I want to try the beta version.' ) }</span>
 						</FormLabel>
 					</FormFieldset>
+					{ showOptOutSurvey && (
+						<FormFieldset className="account__hosting-dashboard-notice-banner">
+							<NoticeBanner
```

Comment:
The NoticeBanner UI looks a bit rough... could we consider using a Banner instead?

Reply:
WFM!

Reply:
Is it possible to just use the new notice 🤔